server/src/domain/auth/auth.service.ts

```diff
@@ -251,11 +251,26 @@ export class AuthService {
         storageLabel = null;
       }
 
+      let defaultQuota: number | null = config.oauth.defaultStorageQuota as keyof OAuthProfile as number;
+      if (typeof defaultQuota !== 'number') {
+        defaultQuota = null;
+      }
```

Comment:
I don't think we need any additional type checking. It should already be valid.

Reply:
I think this can just be `const defaultQuora = config.oauth.defaultStorageQuota`